parameter_files/TestParsing/WrapJSonTestMod.F90

```diff
@@ -0,0 +1,8 @@
+module shr_sys_mod
+
+contains
+
+  subroutine shr_sys_abort()
+    stop
```

Comment:
With a stop, you won't be able to test error handling. This is where a unittest framework can really help.